ai-ml/trainium-inferentia/eks.tf

```diff
@@ -121,6 +121,7 @@ module "eks" {
       labels = {
         WorkerType    = "ON_DEMAND"
         NodeGroupType = "core"
+        workload      = "rayhead"
```

Comment:
Its not advised to run on core node group. rayhead can be run on Karpenter node as well

Reply:
Once Karpenter integration is working and fully tested, will remove this.